packages/ui-components/src/components/Map/LeafletStyles.js

```diff
@@ -59,4 +59,10 @@ export const LeafletStyles = css`
   .leaflet-tooltip-right:before {
     border-right-color: #555;
   }
+  .leaflet-bottom.leaflet-right .leaflet-control-attribution {
+    display: none;
+  }
+  .leaflet-control.leaflet-control-attribution {
+    background: none;
+  }
```

Comment:
Can you add a comment around this stuff to indicate that we're hiding it because we manually display it on the left instead? It just looks a bit dodgy in our central Map styles otherwise